src/xcb_util.hpp

```diff
@@ -80,6 +81,13 @@ std::vector<uint32_t> get_property_value (xcb_connection_t *c,
  */
 std::vector<monitor_info> get_monitors (xcb_connection_t *c, xcb_window_t root);
 
+/**
+ * Get crtc of the primary monitor.
+ *
+ * NOTE: Returned monitor_info.name is unsed as it is unused.
```

Comment:
it's unset (not unsed) i believe. I guess it's better rto fix it here and in a couple more places

Reply:
wdym?

Reply:
fixed